handlers/handler.go

```diff
@@ -82,12 +85,20 @@ func InitHandlers(e *echo.Echo) error {
 	e.POST(p+"secret", SetSecret)
 	e.PUT(p+"secret/update", SetSecret)
 
+	// Worker
+	e.GET(p+"worker/secret", GetWorkerRegisterSecret)
+	e.POST(p+"worker/register", RegisterWorker)
```

Comment:
To follow some practices, this could be just simply `worker` and a `POST` since GET `worker` gets a worker. I don't think you need the `register` part.

Reply:
Good catch. Thanks!